pkgs/development/tools/github/github-release/default.nix

```diff
@@ -7,7 +7,7 @@ let
   metadata = assert linuxPredicate || bsdPredicate || darwinPredicate;
     if linuxPredicate then
       { arch = "linux-amd64";
-        sha256 = "0b3h0d0qsrjx99kcd2cf71xijh44wm5rpm2sr54snh3f7macj2p1";
+        sha256 = "0p0qj911nmmdj0r7wx3363gid8g4bm3my6mj3d6s4mwgh9lfisiz";
         archiveBinaryPath = "linux/amd64"; }
     else if bsdPredicate then
       { arch = "freebsd-amd64";
```

Comment:
Hmm there are two more hashes. Can you check those out?

Reply:
This is an example of something my bot is too dumb to figure out. I'll add this package to the blacklist.